### DISCUSSION TITLE
悪魔領域の魔法強化案

### DISCUSSION THREAD

**Discussion by tanguband** [hengband/hengband#4325] [Ideas]
### 目的
攻撃魔法4領域のなかで最弱の悪魔を選ばれる魅力のある攻撃領域に強化する

### 現状分析と改善案
カオスは魔力炸裂、カオスボルト、破壊、魔力の嵐
破邪は聖なる光球、ハルマゲドン、神威
暗黒は吸血の連矢、地獄の劫火、幽体化
どの領域も特徴となる魔法がありそれが魅力につながっている
悪魔といえばコレと言われるような特徴的な魔法を導入したい

攻撃のメイン属性は火、地獄、プラズマなので、現在のフレーバーを壊さないように属性は引き継ぎたい

補助魔法は充実している、デュアルを対象に不利にならないような調整を行いたい

自分が実際に触った感触と溜まり場、wikiなどの意見では攻撃魔法のコスパが悪いので改善したい

### 魔法変更案

地獄の矢　-> ファイア・ボルト
マジックミサイルの次の魔法になるが、この段階だと地獄属性は若干使いづらい
次の変更とも関連するが序盤に使いやすいファイア・ボルトに変更する
ダイスは(6+(レベル-5)/4)d8のままとしてカオスのファイア・ボルトより弱いので消費MPを減らす

地獄の焔　->　地獄球
属性を劫火から地獄に変更するがダメージを現状の2倍程度にする
ダメージ 6d6+レベルx2 半径2(レベル30以上で3) の地獄の球
メイジ、ハイメイジ（プリースト？）はダメージ 6d6+レベルx3
ダメージ2倍は強すぎるように思うかもしれないが地獄領域は邪悪半減、アンデッド無効
比較として聖なる光球は邪悪2倍、善良無効
beta14でのモンスターを大まかに調べてみると
総数1361　邪悪743　非邪悪かつ地獄耐性なし528　アンデッド143　善良87
地獄球のフルパワーが通るのは528に対して同じ威力となる聖なる光球の2倍は743
地獄球が無効は143に対して聖なる光球は87
比較しても強すぎることはなく聖なる光球の対となる魅力のある魔法になる
問題点はスペルマスターだとすべての敵に効果的な攻撃ができてしまうので職業別に威力を調整する

ファイア・ボール
悪魔の得意魔法として威力を強化
ダメージ(レベル+55)　->　ダメージ(レベル+65)

地獄球　->　悪魔の目
テレパシーを3冊目から2冊目に移動、習得LVを30前後で調整する

悪魔の目　->　地獄の焔
リストラ

地獄の波動　->　地獄嵐
地獄嵐を4冊目から3冊目に移動、魔法職のLV40台の攻撃力不足を解消する
スペルマスターには強すぎる可能性があるため習得LVで調整する

地獄嵐　->　地獄の波動
地獄の波動を3冊目に移動
4冊目にふさわしく威力を3倍に強化、習得LVを上げる
強力になるが4冊目、習得LVの関係で使いづらいかもしれない
地獄嵐の入れ替えのための移動なので実用性は考慮しない

以上が強化案の原案になります
基本は攻撃魔法強化でハイメイジ、メイジを念頭に置いています
プリースト、混沌の戦士、その他デュアル、スペルマスターは威力や習得LVで調整します
開発者やプレイヤーの意見があれば取り入れて調整したいと思います
自分の力量で変更できる内容であれば開発者の許可がおりたらプルリクエストを作成したいと思います

ご意見、アイデアなどありましたら、よろしくお願いします

**Comment by tanguband** [comment]
discordでいただいた意見をもとに再度調整しました

地獄の矢　-> ファイア・ボルト
前回同様

地獄の焔　->　地獄球
スペルマスターをあえて除外する必要はないとの意見があったので
ダメージ 6d6+レベル＊2 半径2(レベル30以上で3) の地獄の球
メイジ、ハイメイジ、スペルマスターはダメージ 6d6+レベル＊3

ファイア・ボールは現状のまま変更しない

地獄球　->　猛火の矢（仮）
ダメージ(レベル＊5)の火炎の矢
破邪魔法の裁きの雷の火炎版
火炎属性は電撃属性より通りが悪いので裁きの雷より習得LVは1LV、消費MPは2点、それぞれ減らす

悪魔の目は職種ごとの影響が大きいため現状のまま変更しない

地獄の波動　->　火炎嵐（仮）
ダメージ 200+レベル＊2 , 半径4 の火炎の球
習得LVと消費MPを暗黒魔法の吸血の連矢と同じにして悪魔領域の得意魔法に位置づけたい

サキュバスの接吻は難度を暗黒の嵐やスターバーストと同様に合わせる

士気高揚　->　猛火の嵐（仮）
ダメージ レベル＊15 , 半径 レベル/5 の火炎の球
地獄嵐を基準として習得LVは1LV早めに消費MPは同じにする
開発の許可が出ればダメージはレベル＊20まで強化したい

ご意見、アイデアなどありましたら、よろしくお願いします

**Comment by tanguband** [comment]
discordでいただいた意見をもとに再度調整しました

地獄の矢　->　ファイア・ボルト
ダメージ(6+(レベル-5)/4)d8　消費MPはカオスのファイアボルトより1減らす

地獄の焔　->　地獄球
ダメージ 6d6+レベル*2 半径2(レベル30以上で3) の地獄の球
メイジ、ハイメイジ、スペルマスターはダメージ 6d6+レベル*3

デーモン支配はリストラ

ファイア・ボールは変更しない

地獄球　->　猛火の矢（仮）
ダメージ(レベル*5)の火炎の矢
裁きの雷より習得LVは1LV、消費MPは2点、それぞれ減らす

悪魔の目は変更しない

地獄の波動は2冊目の一番最後に移動　習得LV、難度はそのまま、消費MPは半減　

火炎嵐（仮）
ダメージ 200+レベル*2 , 半径4 の火炎の球
習得LVと消費MPを暗黒魔法の吸血の連矢と同じ

サキュバスの接吻は難度を暗黒の嵐やスターバーストと同様に合わせる

士気高揚　->　猛火の嵐（仮）
ダメージ レベル*15 , 半径 レベル/5 の火炎の球
地獄嵐を基準として習得LVは1LV早めに消費MP、難度は同じにする

ご意見、アイデアなどありましたら、よろしくお願いします

**Comment by meetname** [comment]
地獄の焔は実質HELL_FIRE属性で善良倍打のあり耐性で軽減されない万能属性ですが、
地獄球は地獄耐性で半減以上、相手が邪悪なだけで半減されてしまう地獄属性では威力が盛られたとしても代わりになるとは思えません。
威力が盛られるといっても所詮一冊目の範疇を出ない範囲でしょうし。

士気高揚はヒーロー付与の他、微回復魔法としても利用可能で余ったMPをHPに転換、そして薬を大量摂取による食べ過ぎ（それに伴う減速）を防ぐことができる非常に有用な魔法です。
また、一領域で祝福とヒーローが使用可能というデュアル向けな魔法が多い悪魔領域の強さと魅力を損なうものと考えます。

強化案なのになぜ悪魔領域の中でも強いほうの魔法を消してしまうのでしょう。

**Comment by tanguband** [comment]
地獄球は半減されてなお地獄の焔同等の威力に調整しています
地獄が通らない序盤の敵には変更されたファイアボルトで対応可能と考えています
終盤の地獄持ちには、他の魔法で対応もしくは別の手段を想定しています

士気高揚に関してはフレーバーを検討した結果、4冊目の除外魔法の候補が他になかったためです
薬で代用可能と判断しました

強化案は言葉足らずでした
正しくは攻撃魔法強化を想定しています

個人的には今まで出来ていたことができなくなることはストレスになるのは同意します

純魔法職での悪魔領域の利用が少なく、何か魅力的なテコ入れができればと考え提案しました

実験的に一旦受け入れていただいくわけにはいかないでしょうか？

他の方の使用感など聞いたたうえで問題があるようでしたら、変更も検討したいと考えています
（正確には修正を開発者に認めてもらわないといけませんが）


私の立場は開発者ではなく協力者であり今回の変更の責任は私にあります

ご意見は私にいただけると助かります

よろしくお願いします

**Comment by RRyukkuriRyu** [comment]
同じ４冊目の呪文であるペット爆破が非常に弱い性能かつ、フレーバー的にも存在感があまりなさそうなので、こちらを削除して士気高揚を残すというのはどうでしょうか？
これなら攻撃呪文と補助呪文の数が変わらないので、デュアルクラスでの使い勝手を大きく変えることなく、純魔法職としての悪魔領域の強化にもなると思います。

**Comment by tanguband** [comment]
議論の結果、士気高揚は場合により魔法での代替が不可能であるとの結論になりました

1冊目の恐慌の代わりに士気高揚を採用します

以前と比較して習得レベルは若干上がる可能性がありますが、
4冊目を拾えるレベルよりは確実に低いため実質的にデュアル職の強化につながると考えます

デュアル視点ですと地獄の焔の変更が弱体化につながる可能性がありますが
ファイアボルトと士気高揚の1冊目採用で相殺または強化になっていると結論しました

問題がなければこちらの仕様が次回Betaで採用される予定です

あとは実際にプレイしていただいて以前と比較して弱体化を感じるようでしたらご意見いただけると助かります

よろしくお願いします

**Comment by tanguband** [comment]
Developブランチで採用された悪魔魔法領域の変更をまとめました
リリースまでに調整されて変更する可能性があります

https://tanguband.github.io/demon.html

あとは実際にプレイしていただいて以前と比較して弱体化を感じるようでしたらご意見いただけると助かります
よろしくお願いします

**Comment by tanguband** [comment]
リリース版 3.0.1.16-Beta で正式採用となりました
変愚蛮怒wiki こと Hengband Spoiler の悪魔領域関連ページは更新してあります

あとは実際にプレイしていただいて以前と比較して弱体化を感じるようでしたらご意見いただけると助かります
よろしくお願いします

**Comment by Ghotiolos** [comment]
当実装は従前よりの欠点の改善には繋がっておらず、そのため攻撃領域として魅力が増したように思えない、というのが率直な感想です。

火炎耐性をもつ邪悪な敵への主な対処手段が長期間プラズマ属性に限られ、なおかつ、プラズマボルトのコストパフォーマンスが悪く、さらには、プラズマ球の高い難度設定が下位のプラズマボルトへの依存を助長させること。さらには、プラズマ球を実用する頃にはTDが低すぎ型落ちとなること。
同格の最終魔法と比して血の呪いの性能が明確に落ちること。また、それにも関わらずステータス低下やエキサイト・モンスターといった非常に重いデメリットが頻繁に発生するため、同時期の他攻撃領域よりも高い加速・維持装備を揃えた後でなければ実用困難なこと。
こうした悪魔領域の魅力を損なっているポイントもフォローすべきではないでしょうか。

確かに悪魔領域そのものは強化されました。火炎破でＤを狩りやすくなりましたし、これによってサキュバスの接吻の難度を落とすのも楽になりました。今や血の呪いで加速したワイアームがブレスを連打しないことを祈る必要もありません。素晴らしい進歩だと思います。
しかし、目的が「悪魔領域の強化」ではなく「悪魔領域を選ばれる魅力のある攻撃領域に強化する」ことであれば、少なくとも、最終魔法の改善だけは不可欠に思えます。

よろしくご検討ください。

**Comment by tanguband** [comment]
実際にプレイしている方の貴重なご意見ありがとうございます

最終魔法につきましては開発メンバーの方が修正の方向で検討に入ったようです

序盤のプラズマボルトまでのつなぎの件は
個人的にプレイした感想としてはハイメイジですとそこまで気にならず
メイジですと習得レベルの関係でプレイに停滞を感じることがありました

開発メンバーの方が悪魔火の調整も検討されているようなので
このあたりのバランスも含めて検討していただけるように相談したいと思います

よろしくお願いします

**Comment by meetname** [comment]
テストプレイを行い確認しましたが、純魔法職で使う場合他の攻撃領域と比較すると、強化が足りていません。
カオス領域は強力な状態異常と属性の通りやすさ、暗黒領域は火力と吸血効果で格上であっても戦いやすいですが、
悪魔には追加効果のない属性なので火力で勝負するしかありません。
（狂気の円環は四冊目で使用可能になる時期が遅く、その頃には混乱耐性持ちがほとんど。また、混乱させたいならカオス領域でもいい）
しかし、消費MP・習得Lv・難度からして火炎破は暗黒の吸血の連矢に対応する魔法として作られていますが、火力も追加効果も劣ります。

火炎の嵐、地獄の嵐も高Lv帯モンスターの1/3程度しか軽減されずに通らないのでイマイチですが、領域が扱う属性的には仕方がない。
血の呪いもひどいですが、強化が入るようなので省略。

他領域の補助魔法を省いてもこの評価なので、
1.さらに威力を上げる
これ以上威力を上げるのが無理ならば、
2.習得Lvを全体的に大幅に引き下げて早熟の魔法領域にする
くらいやってくれなければ、他の攻撃領域と競える性能ではないと思います。

**Comment by tanguband** [comment]
テストプレイありがとうございます

上のコメントでも書きましたがバランスにつきましては
引き続き開発メンバーと相談したいと思います

よろしくお願いします

**Comment by meetname** [comment]
深淵招来がMP消費に対して威力が低すぎます。カオス領域の魔力の嵐と威力が同じなのに1.53倍もの消費です。
暗黒や破邪の最終攻撃魔法と比較しても群を抜いてMP消費に対する威力が低く、燃費が悪いです。
浮遊倍打が入ったとしても他領域の燃費を下回ります。
メイジやプリーストの場合、第一領域の熟練度達人の深淵招来を使うより第二領域の熟練度エキスパートの魔力の嵐を使ったほうが燃費がいいくらいです。
威力を600点にし、相手を選ぶ代わりに刺さる相手には炎の嵐や地獄嵐と同じく750点でいいと思うのですが。

いくら鈍足が付与できることがあるとしても、ほぼ同じ性能（習得レベルや最終的な失率で負けていますが）の魔力の嵐の属性が深淵になっただけでこのMP消費の増加は多すぎます。

**Reply by dis-** [reply]
燃費の悪さは意図的に設定したものです。
悪魔領域には補助魔法が充実している分、汎用攻撃魔法の純粋スペックでは一段劣るように設計しています。

**Comment by meetname** [comment]
気軽に敵除去できる*破壊*、強力に行動妨害ができるソニックブーム、魔力充填があるカオス、
狂乱戦士による疑似的な加速、悪魔変身に近い性能を持つ吸血鬼変化、実質的にHP2倍化＋壁抜けで魔王変化以上に耐久性能を増やせる幽体化がある暗黒に対して
攻撃魔法にペナルティを受けるほど悪魔領域の補助魔法が強いとは思えません。

**Reply by dis-** [reply]
まず、1-3冊目で攻撃属性が偏って苦労する分4冊目が強力な暗黒領域を引き合いに出すのは違います。
地獄の業火-幽体化は最強ですが、他領域にそれに匹敵するスペックを与えることはありません。

カオス領域との比較になりますが、カオスには便利な攻撃魔法が多く含まれますが、*破壊*以外の補助魔法に特筆すべきものはありません。
邪なる祝福 耐火炎 士気高揚 ビジョン 耐地獄 悪魔の目 悪魔のクローク 悪魔変化
ここまでを1-3冊目に持つ悪魔領域は明確に探索において有利です。デュアルクラスではさらに顕著になります。
特に悪魔の目と悪魔変化は魔法道具による代替ができないこともあり、領域全体の設計に大きく影響があります。

もう一つ比較点として、炎地獄の属性攻撃をかなり優遇していることも考慮に入れるべきです。
火炎破、炎の嵐、地獄嵐により中盤-終盤かなり楽になることを勘定に入れて深淵招来が設定されています。

最後に、これから深淵招来のMP効率を改善する可能性はありますが、威力を安易に上げることはありません。
500点の遠距離攻撃はゲームに勝利するために十分な威力ですが、威力を増すごとに最終戦が明確に簡単になります。
旧血の呪いの600点はペナルティありきの高火力であり、ペナルティなしに同等の火力を与えることはありません。

**Reply by meetname** [reply]
600点はペナルティありきなのでダメージは上げられないという点は了解しました。
消費MPについては見直される可能性があるとのことで、いったん様子見しようと思います。
